docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -50,6 +50,45 @@ The reconciliation loop for `Module` runs the following steps:
     2. successful build jobs;
     3. successful signing jobs.
 
+## Security and permissions
+
+Loading kernel modules is a highly sensitive operation.
+Once loaded, kernel modules have all possible permissions to do any kind of operation on the node.
+
+### `ServiceAccounts` and `SecurityContextConstraints`
+
+KMM creates privileged workload to load the kernel modules on nodes.
+That workload needs `ServiceAccounts` able to use the `privileged` `SecurityContextConstraint` (SCC).
+
+The authorization model for that workload depends on the `Module`'s namespace, as well as its spec:
+
+- if the `.spec.moduleLoader.serviceAccountName` or `.spec.devicePlugin.serviceAccountName` fields are set, they are
+  always used;
+- if those fields are not set, then:
+    - if the `Module` is created in the operator's namespace (`openshift-kmm` by default), then KMM will use its
+      default, powerful `ServiceAccounts` to run the DaemonSets;
+    - if the `Module` is created in any other namespace, then KMM will run the DaemonSets as the namespace's `default`
+      `ServiceAccount`, which cannot run privileged workload unless you manually allow it to use the `privileged` SCC.
+
+!!! warning "`openshift-kmm` is a trusted namespace"
+
+    When setting up RBAC permissions, keep in mind that any user or ServiceAccount creating a `Module` resource in the
+    `openshift-kmm` namespace will result in KMM automatically running privileged workload on potentially all nodes in
+    cluster.
```

Comment:
If the user set RBAC then it will happen in any namespace no?!

Reply:
Correct. This warning reminds people that they don't need to explicitly set RBAC when creating a `Module` in `openshift-kmm` for the kmod to be loaded everywhere.